packages/plugins/workspace-search/example/node/package.json

```diff
@@ -3,11 +3,10 @@
   "version": "1.0.0",
   "description": "Example of workspace search with node.",
   "scripts": {
-    "preinstall": "cd ../.. && npm install",
     "start": "node --preserve-symlinks index.js"
   },
   "dependencies": {
-    "@blockly/plugin-workspace-search": "file:../..",
```

Comment:
Do we want this to be the published version? If they are a test then we want them to be updated when we change something in the workspace search file.

Reply:
Lerna takes care of symlinking to the local version. Since this is for development, if you change something locally it should be reflected here.